packages/office-addin-dev-settings/README.md

```diff
@@ -256,6 +259,6 @@ Syntax:
 
 `manifest`: path to manifest file. 
 
-`runtime`: Office runtime to load (currently accepts 'ie', 'edge', or 'default'). 
+`runtime`: Office runtime to load (currently accepts 'ie', 'edge', 'edgechromium', or 'default'). 
```

Comment:
What is 'runtime' used for? This is specifying which webview type, not a runtime really. Can we change this to `<web-view-type>`?